ocp_resources/machine_config_pool.py

```diff
@@ -2,8 +2,69 @@
 
 
 class MachineConfigPool(Resource):
+    """
+    MachineConfigPool object. API reference:
+    https://docs.openshift.com/container-platform/4.12/rest_api/machine_apis/machineconfigpool-machineconfiguration-openshift-io-v1.html
+
+    Args:
+        name (string): Object name.
+        labels (dict): Labels by which to match related resource to this MachineConfigPool object.
+        machine_config_selector (object): List of matching labels, to determine on which nodes to apply this
+            MachineConfigPool object.
+        configuration (object): Current MachineConfig object for the machine config pool.
+        yaml_file (str): yaml file for the resource.
+        node_selector_labels (object (string)): Map of {key,value} pairs that determine node selection criteria.
+        max_unavailable (integer/string): Number/percentage of nodes that can go Unavailable during an update.
+        paused (object (boolean)): Whether changes to this MachineConfigPool should be stopped.
+    """
+
     api_group = Resource.ApiGroup.MACHINECONFIGURATION_OPENSHIFT_IO
 
     class Status(Resource.Status):
         UPDATED = "Updated"
         UPDATING = "Updating"
+
+    def __init__(
+        self,
+        name=None,
+        labels=None,
```

Comment:
`labels` should be removed; will be part of `kwargs`

Reply:
Ack